packages/x-components/src/components/__tests__/base-grid.spec.ts

```diff
@@ -101,3 +131,9 @@ interface BaseGridComponentAPI {
   getDefaultSlot: () => WrapperArray<Vue>;
   getScopedSlot: (modelName: string) => WrapperArray<Vue>;
 }
+
+// TODO Remove this interface when the original from the next-queries.list.vue is exported.
+interface NextQueriesGroup extends ListItem {
```

Comment:
Contrary to the `GridItem` type which should be private (just for the file it is being used in), this one should be public, so you can just move it to the utils file or the search-types project instead of duplicating it.